project/Dependencies.scala

```diff
@@ -1067,6 +1068,8 @@ object Dependencies {
 
     if (problems.nonEmpty) {
       problems.foreach(p => log.error(p._2))
+      if(problems.nonEmpty)
```

Comment:
```suggestion
```

Reply:
The common case is that there are no errors when checking the dependency whitelist and this message appears for each project so removing the `if` we'd get:

```
Found 0 errors
Found 0 errors
Found 0 errors
Found 0 errors
Found 0 errors
Found 0 errors
Found 0 errors
Found 0 errors
[error] project-lagom-name Found version x.x.x instead of y.y.y
Found 1 errors
Found 0 errors
Found 0 errors
Found 0 errors
```

With the code as is we get a less verbose:

```
[error] project-lagom-name Found version x.x.x instead of y.y.y
Found 1 errors
```

Reply:
I think we're talking about two different removals.  I'm removing a check that was verified 2 lines above.  Unless `problems` is mutable.

Reply:
🤦🏼‍♂️

You are right